app/views/home/show.html.erb

```diff
@@ -0,0 +1 @@
+  <%= link_to "Sign in with Google", "/auth/google_oauth2" %>
```

Comment:
Just a minor thing, we have a test in master that looks for a button on the home page that says 'Login with Google' so since this is a link and different wording we might want to change that test.

Reply:
Yeah that test is gone for now and will have to be redone since I started from scratch. It will be relatively easy to remake.